### DISCUSSION TITLE
Filament/Form V2: Can form fields have name attribute?

### DISCUSSION THREAD

**Discussion by yob-yob** [filamentphp/filament#671] [Ideas]
I would like to build a form using filament/forms... but the problem is that I don't want to submit the form using Livewire. I wanna submit it using normal Laravel Request that would be handled by a controller.

**Comment by danharrin** [comment]
Sorry, the forms package is tightly integrated with Livewire which allows us to abuse Alpine's `entangle` feature in many places. Most fields, except basic text inputs and selects, would not be suitable for use with controllers.
